posts/flexible-data-model-design.md

```diff
@@ -0,0 +1,349 @@
+---
+authors:
+- Pasha Paterson
+tags:
+- Java
+- data modelling
+- coding standards
+- design
+date: 2019-02-26T18:16:00.000Z
+title: "Flexible Data Model Design"
+image:
+---
+
+The only thing that most discussions of “standards” agree on is that no one agrees on standards. The world of coding standards, in particular, is perhaps one of the most hotly-contested battlegrounds of computer science. Given the patterns and trends emerging in the usage of Java today, there are certain design and implementation practices that, while not providing an end to anyone’s holy wars, do tend to make the most pragmatic sense. This discussion will focus on design principles in one of the most fundamental of these arenas — how to design and implement a data model that can be used flexibly across multiple contexts in a large application architecture.
+
+A lot of the material in this post is pretty basic, so if you're impatient and want to skip to the new stuff, look for this guy: :running:
```

Comment:
Based on rendering in ghost https://test-ippon.ghost.io/ghost, the emoji injection does not work. It will take the utf8 tho 🏃